### ISSUE TITLE
Cannot list-inventory if any are shutdown

### ISSUE THREAD

**Issue by TimOrme** [wagoodman/bridgy#19]
It looks like bridgy relies on PrivateIpAddress being set, which is not always the case for every instance:

https://github.com/wagoodman/bridgy/blob/master/bridgy/inventory/aws.py#L60

Some of my instances return a state of:

```json
"State": {
"Code": 48,
"Name": "terminated"
}
```

And instances of this type do not have a field of PrivateIpAddress, causing the following error when trying to list the inventory:

```python
Traceback (most recent call last):
  File "/home/torme/.local/bin/bridgy", line 11, in <module>
    sys.exit(main())
  File "/home/torme/.local/lib/python2.7/site-packages/bridgy/__main__.py", line 356, in main
    handler(args, config)
  File "/home/torme/.local/lib/python2.7/site-packages/bridgy/utils.py", line 31, in wrapper
    func(*args,**kwargs)
  File "/home/torme/.local/lib/python2.7/site-packages/bridgy/__main__.py", line 258, in list_inventory_handler
    for ip, name, aliases, source in inventory.instances(config):
  File "/home/torme/.local/lib/python2.7/site-packages/bridgy/utils.py", line 45, in __call__
    return self[args]
  File "/home/torme/.local/lib/python2.7/site-packages/bridgy/utils.py", line 47, in __missing__
    ret = self[key] = self.f(*key)
  File "/home/torme/.local/lib/python2.7/site-packages/bridgy/inventory/__init__.py", line 105, in instances
    all_instances = inventory(config).instances()
  File "/home/torme/.local/lib/python2.7/site-packages/bridgy/inventory/source.py", line 88, in instances
    instances.extend(inventory.instances())
  File "/home/torme/.local/lib/python2.7/site-packages/bridgy/inventory/aws.py", line 60, in instances
    elif instance['PrivateIpAddress']:
KeyError: 'PrivateIpAddress'
```

**Comment by wagoodman** [comment]
Yup, that should get fixed. Seeing that this tool is used primarily for getting into servers that are up I'd be inclined to not include any systems on the list that don't have a DNS or IP address... in which case list-inventory is only showing systems that are running and that you may be able to ssh into. What are your thoughts?

**Comment by TimOrme** [comment]
Yeah, I think that makes sense. Given that the primary usage is to access those boxes, I think filtering out anything thats inaccessible makes sense.

I can try and submit a PR if thats helpful?

**Comment by wagoodman** [comment]
Yup, PRs are indeed welcome!

**Comment by wagoodman** [comment]
Closed by PR #20
